src/ui/public/vis/agg_config_result.js

```diff
@@ -3,7 +3,7 @@ define(function () {
 
   function AggConfigResult(aggConfig, parent, value, key) {
     this.key = key;
-    this.value = value;
+    this.value = value === null ? false : value; // restrict to primitive for sorting in Angular
```

Comment:
Oddly enough, for the bug in my use case, whatever value I use to replace here isn't actually visible in the UI.